rclpy/rclpy/qos.py

```diff
@@ -106,18 +106,36 @@ def get_c_qos_profile(self):
 
 
 class QoSHistoryPolicy(IntEnum):
+    """
+    Class defining enums for QoS History settings.
```

Comment:
I would avoid starting with `Class` (similar as what the linters check for avoiding to start with `This ...`). And singular for enum (rather then enums). Suggesting:

> Enum for QoS History settings.

The plural / singular applies to all comments.